docs/user-guide/import.md

```diff
@@ -0,0 +1,53 @@
+# Importing argocd packages 
+
+## Issue 
+When importing argocd packages in your own projects , you may face some erros when downloading the dependencies , such as unknown revision v0.0.0. This is because argocd directly depends on some kubernetes packages which have these unknown v0.0.0 version in their go.mod.
```

Comment:
```suggestion
When importing argocd packages in your own projects, you may face some errors when downloading the dependencies, such as "unknown revision v0.0.0". This is because argocd directly depends on some Kubernetes packages which have these unknown v0.0.0 version in their go.mod.
```